src/hu/elte/txtuml/stdlib/lang/SystemImpl.java

```diff
@@ -0,0 +1,89 @@
+package hu.elte.txtuml.stdlib.lang;
+
+public class SystemImpl implements System{
+
+	@Override
+	public String clearProperty(String key) {
+		return  java.lang.System.clearProperty(key);
+	}
+
+	@Override
+	public String currentTimeMillis() {
+		long  current = java.lang.System.currentTimeMillis();
+		java.lang.System.out.println("original long = " + current);
+		String d = String.valueOf(current);
+		return  d;
```

Comment:
You don't need temporary variables, just put everything together in the `return` statement.